tests/e2e/support/objects/runtime/application.ts

```diff
@@ -1,5 +1,8 @@
 import { Page } from 'playwright'
+import util from 'util'
 
+const appSwitcherButton = '#_appSwitcherButton'
+const appListSelector = `//ul[contains(@class, "applications-list")]//a[@href="#/%s" or @href="/%s"]`
```

Comment:
the selector is for a specific app
```suggestion
const appSelector = `//ul[contains(@class, "applications-list")]//a[@href="#/%s" or @href="/%s"]`
```